teps/0086-changing-the-way-result-parameters-are-stored.md

```diff
@@ -1,13 +1,15 @@
 ---
-status: proposed
+status: implementable
```

Comment:
we shouldn't mark this TEP as implementable because we haven't landed on a solution yet, all the options are alternatives in this TEP and we're experimenting to figure out the way forward

cc @pritidesai

Reply:
thanks @jerop, yup, @Tomcli please update the status back to `proposed`. We have not proposed any solution yet.